packages/playground/vite.config.js

```diff
@@ -14,5 +14,14 @@ export default defineConfig({
     },
     sourcemap: true,
   },
-  plugins: [sentryVitePlugin()],
+  plugins: [
+    sentryVitePlugin({
+      org: "sentry-sdks",
```

Comment:
nit: can we share common settings with other configs? it would be nicer if possible

Reply:
Sure, tbh I'm not really happy with adding actual projects here. I just used this values to test out basic functionality (such as calling the CLI via the plugin). I can extract this to some sort of placeholder config object but I'll try to come up with something different